packages/block-library/src/post-content/edit.js

```diff
@@ -45,12 +46,29 @@ function EditableContent( { context = {} } ) {
 		{ id: postId }
 	);
 
+	const hasInnerBlocks = useSelect(
+		( select ) =>
+			select( blockEditorStore ).getBlock( clientId ).innerBlocks.length >
```

Comment:
`getBlock` can technically return `null`. Would it be worth adding optional chaining? This might be a bit of a theoretical issue, and I'm possibly overly cautious when it comes to what these selectors return! 😄